HaxeManual/13-debugging.tex

```diff
@@ -68,6 +68,12 @@ \section{Tracing Types}
 \end{lstlisting}
 
 
+\section{IDE Integration}
+\label{debugging-ide-integration}
+
+Haxe is supported in a \href{https://haxe.org/documentation/introduction/editors-and-ides.html}{number of editors and IDEs}. Some IDEs, such as Visual Studio Code (with the \href{https://marketplace.visualstudio.com/items?itemName=nadako.vshaxe}{vshaxe} extension), support debugging of Haxe code with breakpoints, stack tracing, etc. Simply run the program in debug mode.

```

Comment:
vshaxe itself only has debugging for eval built-in, so it may be misleading to link to the marketplace page for that. Perhaps https://github.com/vshaxe/vshaxe/wiki/Debugging is a better thing to link to.